.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   build:
     env:
       JRUBY_OPTS: '-J-XX:+TieredCompilation -J-XX:TieredStopAtLevel=1 -J-Xss2m -Xcompile.invokedynamic=false'
+      JAVA_OPTS: '-Djava.security.properties=docker/ecco-java.security' # Enable TLS 1.1, required by MySQL 5.7
```

Comment:
can you "tie" an ENV to a specific run in the matrix (this is a leading question, as I think you can) if so we could just do this for the MySQL 5.7 run

Reply:
I can't find anything about this in the docs actually :/

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-environment-variables-in-a-matrix

Reply:
I think/hope `JAVA_OPTS: ''` works as that would be it for the 8.0 case.

Reply:
> I think/hope `JAVA_OPTS: ''` works as that would be it for the 8.0 case.

It works! 🎉   030c612